package.json

```diff
@@ -5,15 +5,16 @@
   "version": "0.1.0",
   "author": "@medusajs",
   "resolutions": {
-    "devcert": "1.2.0"
+    "devcert": "1.2.0",
+    "postcss": "^8.4.5"
   },
   "dependencies": {
     "@emotion/core": "^10.0.28",
     "@emotion/styled": "^10.0.27",
     "@popperjs/core": "^2.10.2",
     "@radix-ui/react-dialog": "^0.1.5",
     "@rebass/forms": "^4.0.6",
-    "@storybook/theming": "^5.3.19",
+    "@storybook/theming": "^6.4.9",
```

Comment:
should this be a dev-dependency?

Reply:
Yea, I think it should be. Will change it in a sec 😄